package.json

```diff
@@ -7,11 +7,15 @@
     "dev": "vite",
     "build": "tsc && NODE_OPTIONS=--max-old-space-size=4096 vite build",
     "lint": "eslint .",
+    "lint:fix": "eslint --fix .",
```

Comment:
consider removing. It's easy enough to run `eslint --fix .` and it's never something CI has to do.

I'd like to get rid of it everywhere

Reply:
I think it'd have to be `yarn run eslint --fix .` right? Removed though.